src/compiler.py

```diff
@@ -961,7 +951,10 @@ def compile_bql(self, bdb, bql, out):
         assert ast.is_bql(bql)
         generator_id = self.generator_id
         if isinstance(bql, ast.ExpBQLProb):
-            raise BQLError(bdb, 'Probability of value is 1-row function.')
+            compile_pdf_joint(bdb, generator_id, self.modelno, bql.targets,
+                bql.constraints, self, out)
+        elif isinstance(bql, ast.ExpBQLProbFn):
+            raise BQLError(bdb, 'Probability of value is 1-column function.')
```

Comment:
Here and below, and in similar places, please consider improving the legibility of the message, as requested above?